source/guides/rapids-docker-with-cuda.md

```diff
@@ -0,0 +1,369 @@
+- **For pip installations**: You need the components from the NVIDIA `runtime` CUDA images
+
+## Understanding CUDA Image Components
+
+NVIDIA provides three tiers of CUDA container images, each building on the previous:
+
+### Base Components (Required for RAPIDS on conda)
+
+The **base** images provide the minimal CUDA runtime environment:
+
+| Component          | Package Name  | Purpose                                           |
+| ------------------ | ------------- | ------------------------------------------------- |
+| CUDA Runtime       | `cuda-cudart` | Core CUDA runtime library (`libcudart.so`)        |
+| CUDA Compatibility | `cuda-compat` | Forward compatibility libraries for older drivers |
+
+### Runtime Components (Required for RAPIDS on pip)
+
+The **runtime** images include all base components plus additional CUDA libraries:
+
+| Component                     | Package Name     | Purpose                                            |
+| ----------------------------- | ---------------- | -------------------------------------------------- |
+| **All Base Components**       | (see above)      | Core CUDA runtime                                  |
+| CUDA Libraries                | `cuda-libraries` | Comprehensive CUDA library collection              |
+| CUDA Math Libraries           | `libcublas`      | Basic Linear Algebra Subprograms (BLAS)            |
+| NVIDIA Performance Primitives | `libnpp`         | Image, signal and video processing primitives      |
+| Sparse Matrix Library         | `libcusparse`    | Sparse matrix operations                           |
+| Profiling Tools               | `cuda-nvtx`      | NVIDIA Tools Extension for profiling               |
+| Communication Library         | `libnccl2`       | Multi-GPU and multi-node collective communications |
+
+### Development Components (Optional)
+
+The **devel** images add development tools to runtime images:
+
+- CUDA development headers and static libraries
+- CUDA compiler (`nvcc`)
+- Debugger and profiler tools
+- Additional development utilities
+
+```{note}
+Development components are typically not needed for RAPIDS usage unless you plan to compile CUDA code within your container.
+```
+
+## Getting the Right Components for Your Setup
+
+The [NVIDIA CUDA Container Images repository](https://gitlab.com/nvidia/container-images/cuda) contains a `dist/` directory with pre-built Dockerfiles organized by CUDA version, Linux distribution, and container type (base, runtime, devel).
+
+### Supported Distributions
+
+**Debian/Ubuntu-based (uses `apt`):**
+
+- **Ubuntu**: `ubuntu2004/`, `ubuntu2204/`, `ubuntu2404/`
+
+**RHEL/CentOS-based (uses `yum`/`dnf`):**
+
+- **Rocky Linux**: `rockylinux8/`, `rockylinux9/`, `rockylinux10/`
+- **RHEL/CentOS**: `ubi8/`, `ubi9/`, `ubi10/`
+- **Amazon Linux**: `amzn2023/`
+- **Azure Linux**: `azl3/`
+- **OpenSUSE**: `opensuse15/` (CUDA 13.0+ only)
```

Comment:
Again I'm worried this will go stale. I would reduce the wording to make it more illustrative. We support the most popular distros, but this is an evolving list, check out the repo for more info.